mods/tdo/metrics.yaml

```diff
@@ -17,4 +17,23 @@ Metrics:
 	ClickDisabledSound: ClickDisabledSound
 	ClickSound: ClickSound
 	ChatMessageColor: FFFFFF
-	SystemMessageColor: FFFF00
\ No newline at end of file
+	SystemMessageColor: FFFF00
+	IncompatibleProtectedGameColor: B22222
+	IncompatibleVersionColor: D3D3D3
+	TextfieldColorHighlight: 562020
+	NormalSelectionColor: FFFFFF
+	AltSelectionColor: 00FFFF
+	CtrlSelectionColor: FFFF00
+
```

Comment:
Is this intentional?

Reply:
No, the only colors I use are variations of green